### DISCUSSION TITLE
Pháp Sư Tiễn Táng Frieren - SuicaoDex

### DISCUSSION THREAD

**Discussion by giscus** [TNTKien/suicaodex#10] [Manga]
# Pháp Sư Tiễn Táng Frieren - SuicaoDex

Tổ đội anh hùng đã đánh bại được quỷ vương và kết thúc cuộc hành trình của họ. Nhưng thế chưa phải là hết, cuộc đời của cô nàng pháp sư Elf này sẽ còn rất dài, hơn cả những người đồng đội cũ của cô, một cuộc phiêu lưu mới để cô trải qua nhiều cung bậc cảm xúc, cũng như là học hỏi thêm về con người.

https://suicaodex.com/manga/b0b721ff-c388-4486-aa0f-c2b0bb321512

**Comment by suicaodeptrai** [comment]
=w=
